autopush/tests/test_endpoint.py

```diff
@@ -1194,16 +1194,12 @@ def _assert_error_response(self, result):
 
     def test_padding(self):
         # Some values can't be padded and still decode.
-        eq_(utils.fix_padding("a"), "a===")
-        self.assertRaises(TypeError,
-                          base64.urlsafe_b64decode,
-                          utils.fix_padding("a"))
-        eq_(utils.fix_padding("ab"), "ab==")
-        base64.urlsafe_b64decode(utils.fix_padding("ab"))
-        eq_(utils.fix_padding("abc"), "abc=")
-        base64.urlsafe_b64decode(utils.fix_padding("abc"))
-        eq_(utils.fix_padding("abcd"), "abcd")
-        base64.urlsafe_b64decode(utils.fix_padding("abcd"))
+        assert_raises(TypeError,
+                      utils.base64url_decode,
+                      "a===")
+        eq_(utils.base64url_decode("ab=="), "\x69")
```

Comment:
I'm happy to change the test to use different vectors; I just used these because they were already there. It's more about laziness than efficiency. :wink: